src/components/Auth/Login.tsx

```diff
@@ -34,22 +37,31 @@ const Login: React.FC = () => {
 
   const onSubmit = async (data: LoginFormValues) => {
     try {
-      const response = await axiosInstance.post('/users/login', data);
-      const token = response.data.access_token;
+      if (sudo) {
+        const response = await axiosInstance.post(`/users/token/sudo`, data);
+        const token = response.data.sudo_token;
 
-      // Store token in localStorage
-      localStorage.setItem('access_token', token);
-      showSnackbar('Login successful!', 'success');
-      dispatch(
-        setUser({
-          email: response.data.email,
-          username: response.data.username,
-          id: response.data.id,
-          profilePicture: response.data.profile_picture_url,
-        })
-      );
-      dispatch(closeModal());
-      navigate('/home');
+        // Store token in localStorage
+        localStorage.setItem('sudo_token', token);
+        navigate('/profile/security');
```

Comment:
make page redirection dynamic, will need to get sudo token from other pages as well